src/main/java/suppliers/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package suppliers;
+
+import core.basesyntax.model.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length - 1);
```

Comment:
давай винесемо змінну Random на рівень класу, щоб не створювати щоразу новий екземпляр, коли ми викликаємо метод getRandomColor()
```suggestion
        int index = new Random().nextInt(Color.values().length);
```

Reply:
Done